cost-analyzer/templates/kubecost-cluster-controller-template.yaml

```diff
@@ -257,6 +257,8 @@ spec:
         - name: http-server
           containerPort: 9731
           hostPort: 9731
+        resources:
+          {{- toYaml .Values.clusterController.resources | nindent 12 }}
```

Comment:
Actually, shouldn't this be `.spec.resources`? Not `.spec.containers[0].resources`?

Reply:
Oops, nvm! https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/